darwin/importer/importer.py

```diff
@@ -1031,11 +1062,8 @@ def process_local_file(local_path):
                 except Exception as exc:
                     console.print(f"Generated an exception: {exc}", style="error")
     else:
-        for local_path in tqdm(
-            {local_file.path for local_file in local_files},
-            desc="Processing local annotation files",
-        ):
-            process_local_file(local_path)
+        for local_file in tqdm(local_files, desc="Processing local annotation files"):
+            process_local_file(local_file)
```

Comment:
This is a change I'd like someone to review. Currently, the importer reads local annotation files in twice:
- [Once early on](https://github.com/v7labs/darwin-py/blob/f974484ca2d96e4aa2bbb2d3042476d4e59b3085/darwin/importer/importer.py#L791) in the import flow to perform validation of the contents of the files
- [A second time](https://github.com/v7labs/darwin-py/blob/f974484ca2d96e4aa2bbb2d3042476d4e59b3085/darwin/importer/importer.py#L953) just before import to send the annotation data

I can't figure out why we would do this, even though [it was explicitly commented here](https://github.com/v7labs/darwin-py/blob/v0.8.10/darwin/importer/importer.py#L356) previously. I don't believe it would help with memory, since we hold all the annotations in memory after the 1st read anyway.

This change means we read in the annotations only once at the beginning, then pass them all the way through to the stage where we're sending import requests. This is useful because the slot <-> annotation validation this PR introduces needs to make changes to the slots of some annotations. If we do this early on and then read the files in again later, the changes we made don't persist. With this change, they do persist